rspec-rerun.gemspec

```diff
@@ -15,6 +15,6 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'rake'
   s.add_development_dependency 'bundler'
-  s.add_development_dependency 'rubocop', '0.29.1'
+  s.add_development_dependency 'rubocop'
```

Comment:
Because no Gemfile.lock is checked in you want to lock this one down, or you'll constantly be having new build failures because Rubocop is being upgraded.